docs/content/blog/authelia-traefik-setup-guide/index.md

```diff
@@ -78,19 +96,29 @@ services:
     secrets:
       - cloudflare_email
       - cloudflare_api_key
+    labels:
+      traefik.enable: 'true'
+      traefik.http.routers.dashboard.rule: 'Host(`traefik.{{< sitevar name="domain" nojs="example.com" >}})'
+      traefik.http.routers.dashboard.entrypoints: 'https'
+      traefik.http.routers.dashboard.middlewares: 'authelia@docker'
+      traefik.http.routers.dashboard.service: 'api@internal'
+      traefik.http.routers.dashboard.tls.certresolver: 'cloudflare'
+      traefik.http.middlewares.authelia.forwardAuth.address: '{{< sitevar name="tls" nojs="http" >}}://{{< sitevar name="host" nojs="authelia" >}}:{{< sitevar name="port" nojs="9091" >}}/api/authz/forward-auth'
+      traefik.http.middlewares.authelia.forwardAuth.trustForwardHeader: 'true'
+      traefik.http.middlewares.authelia.forwardAuth.authResponseHeaders: 'Remote-User,Remote-Groups,Remote-Name,Remote-Email'
 
   whoami:
     image: traefik/whoami
     restart: unless-stopped
     container_name: whoami
     labels:
-      - traefik.enable=true
-      - traefik.http.routers.whoami.rule=Host(`whoami.example.com`)
-      - traefik.http.routers.whoami.entrypoints=https
-      - traefik.http.routers.whoami.tls=true
-      - traefik.http.services.whoami.loadbalancer.server.port=80
+      traefik.enable: 'true'
+      traefik.http.routers.whoami.rule: Host(`whoami.{{< sitevar name="domain" nojs="example.com" >}}`)
+      traefik.http.routers.whoami.entrypoints: 'https'
+      traefik.http.services.whoami.loadbalancer.server.port: '80'
```

Comment:
```suggestion
```

This isn't necessary as the container only has a single exposed port, Traefik will pick this up by default.